userScript.py

```diff
@@ -0,0 +1,47 @@
+import json
+import requests
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+BASE_URL = os.environ.get('SERVER_HOST')
+local = "/Users/dviravraham/PycharmProjects/final-exercise-DvirAvraham14/pres.pptx"
```

Comment:
Better to put the test file inside your project so that you'd be able to run it from every computer and every location.